examples/src/main/java/com/flipkart/flux/examples/benchmark/RunBenchMarkWorkFlowA.java

```diff
@@ -19,7 +19,7 @@ public static void main(String args[]) {
 
         /* Lets invoke our workflow */
         System.out.println("[Main] Starting BenchMarkWorkFlowA  execution");
-        for (int i = 1; i <= 100 ; i++) {
+        for (int i = 1; i <= 1 ; i++) {
```

Comment:
why 1? Why is this reduced to 1?

Reply:
This is a helper class to generate few workflows at local.